### DISCUSSION TITLE
Is it possible to cascade mcp23008?

### DISCUSSION THREAD

**Discussion by simone1970** [arendst/Tasmota#17758] [Q&A] [answered]
Good morning everyone...
I wanted to ask if it was possible to add more mcp23008 to be able to expand to 16-24-32 gpio
I couldn't find anything about it....
Thank you

**Comment by barbudor** [comment, marked as answer]
Unfortunately not as of today. The driver supports only 1 device MCP23008 (8 i/o) or 1 MCP23017 (16 i/o) 
You can have up to 4x PCF8574 or 4x 74HC595 if you want more.
Tasmota is however limites to 32 POWER controls

**Reply by simone1970** [reply]
Thank you ...
